test.sh

```diff
@@ -42,3 +42,6 @@ if [[ "${JOB_TYPE}" != "local" ]]; then
   chmod +x codecov
   ./codecov -t ${CODECOV_TOKEN} -r "openshift/dynamic-plugin-sdk" ${REF_FLAGS} --dir ./coverage
 fi
+
+# Publish
+./publish.sh
```

Comment:
I use `./test.sh` to verify everything... will this will prompt a publish to npm? I don't have access, but will it ask me for npm creds?

Reply:
Agreed. I can make these updates. I was primarily focused on the path to publishing first.

Reply:
Same as @andrewballantyne - I also use `test.sh` to verify my changes locally, the publish can be triggered when we are in CI env.

Reply:
Updated so this script will not impact local runs of `./test.sh` (aside from a single line of output: `skipping publish npm packages`)